reVX/least_cost_xmission/config/config.json

```diff
@@ -0,0 +1 @@
+{"iso_lookup": {"MISO": 3, "SCE": 2, "Southeast": 4, "TEPPC": 1}, "min_power_classes": {"100MW": 0, "200MW": 100, "400MW": 200, "1000MW": 400}, "new_substation_costs": {"MISO": {"138": 7800000, "230": 9700000, "500": 20300000, "69": 5700000}, "SCE": {"138": 8870000, "230": 17525000, "500": 37017000, "69": 3975000}, "Southeast": {"138": 4692000, "230": 5732000, "500": 20364000, "69": 3064000}, "TEPPC": {"138": 5541000, "230": 9171000, "500": 15489000, "69": 3498000}}, "power_classes": {"1000MW": 1500, "100MW": 102, "200MW": 205, "400MW": 400}, "transformer_costs": {"115": {"115": 5230, "138": 4260, "161": 4480, "230": 4970, "345": 5790, "500": 7110, "69": 3840}, "138": {"115": 4260, "138": 5790, "161": 4720, "230": 4970, "345": 5790, "500": 7110, "69": 4260}, "161": {"115": 4480, "138": 4720, "161": 6420, "230": 5230, "345": 6100, "500": 7490, "69": 4480}, "230": {"115": 4970, "138": 4970, "161": 5230, "230": 7110, "345": 6100, "500": 7490, "69": 4970}, "345": {"115": 5790, "138": 5790, "161": 6100, "230": 6100, "345": 8670, "500": 7860, "69": 6100}, "500": {"115": 7110, "138": 7110, "161": 7490, "230": 7490, "345": 7860, "500": 11610, "69": 7860}, "69": {"115": 3840, "138": 4260, "161": 4480, "230": 4970, "345": 6100, "500": 7860, "69": 4720}}, "power_to_voltage": {"102": 69, "1500": 500, "171": 115, "205": 138, "240": 161, "343": 230, "400": 230, "68": 46, "750": 345}, "upgrade_substation_costs": {"MISO": {"138": 1600000, "230": 2200000, "500": 5300000, "69": 1100000}, "SCE": {"138": 1117000, "230": 3975000, "500": 9665000, "69": 767000}, "Southeast": {"138": 1179000, "230": 2210000, "500": 8340000, "69": 917000}, "TEPPC": {"138": 2198000, "230": 5500000, "500": 10236000, "69": 1352000}}, "iso_multipliers": [{"iso": "TEPPC", "land_use": {"cropland": 1, "forest": 2.25, "suburban": 1.3, "urban": 1.59, "wetland": 1.2}, "slope": {"hill_mult": 1.4, "hill_slope": 2, "mtn_mult": 1.75, "mtn_slope": 8}}, {"iso": "SCE", "land_use": {"cropland": 1, "forest": 3, "suburban": 2, "urban": 3, "wetland": 2}, "slope": {"hill_mult": 1.5, "hill_slope": 2, "mtn_mult": 2, "mtn_slope": 8}}, {"iso": "MISO", "land_use": {"cropland": 1.03, "forest": 1.2, "suburban": 1.08, "urban": 1.2, "wetland": 1.8}, "slope": {"hill_mult": 1.1, "hill_slope": 2, "mtn_mult": 1.21, "mtn_slope": 8}}, {"iso": "Southeast", "land_use": {"cropland": 1, "forest": 1.47, "suburban": 1.75, "urban": 1.12, "wetland": 1.33}, "slope": {"hill_mult": 1.2, "hill_slope": 2, "mtn_mult": 1.6, "mtn_slope": 8}}], "base_line_costs": {"MISO": {"102": 1255000, "1500": 2787000, "205": 1446000, "400": 1695000}, "SCE": {"102": 1524000, "1500": 4777000, "205": 2084000, "400": 3034000}, "Southeast": {"102": 819000, "1500": 4056000, "205": 984000, "400": 1568000}, "TEPPC": {"102": 984000, "1500": 2248000, "205": 1180000, "400": 1570000}}, "nlcd_land_use_classes": {"cropland": [80, 81], "forest": [41, 42, 43], "wetland": [90, 95], "suburban": [21, 22, 23], "urban": [24]}}
```

Comment:
Is it safe to assume github actions will format that?

Reply:
Yeah I did a no-verify commit just to get up before calling it a day. Going to finish it up / clean it up now.